lib/membrane_mp4/muxer/movie_box.ex

```diff
@@ -0,0 +1,395 @@
+defmodule Membrane.MP4.Muxer.MovieBox do
+  @moduledoc """
+  This module cotainins a set of utilities for creating an MPEG-4 movie box.
```

Comment:
It would be best to include the docs about MovieBox, what it is, what it does, how it looks. The bare minimum would be to include the links to the RFC/standard with that description